src/main/java/com/google/impactdashboard/configuration/Constants.java

```diff
@@ -0,0 +1,58 @@
+package com.google.impactdashboard.configuration;
+
+/** A class to hold constants used throughout the project. */
+public class Constants {
+
+  /** The project id of the project being used to deploy this webapp. */
+  public static final String PROJECT_ID = "concord-intern";
+
+  /** The path to the service account key. */
+  public static final String PATH_TO_SERVICE_ACCOUNT_KEY = 
+    "/usr/local/google/home/carolinelui/Downloads/concord-intern-10ce059284ad.json";
+
+  /** The name of the table holding recommendations data. */
+  public static final String RECOMMENDATIONS_TABLE = "Recommendations";
+
+  /** The name of the table holding Iam Bindings data. */
+  public static final String IAM_BINDINGS_TABLE = "IAM_Bindings";
+
+  /** The name of the database holding the test data. */
+  public static final String TEST_DATABASE = "Test_Capstone";
+
+  /** The name of the database holding the real data. */
+  public static final String DATABASE = "";
+
+  /* Column names of Recommendations Table. */
+
+  /** The name of the column holding the project id. */
+  public static final String PROJECT_ID_COLUMN_RECOMMENDATIONS = "ProjectId";
```

Comment:
Nit, can move "RECOMMENDATIONS" to the beginning to keep the _COLUMN naming pattern.

Reply:
Good idea! I'll fix this for all the columns!